src/main/java/gyro/google/compute/InstanceGroupResource.java

```diff
@@ -238,6 +294,55 @@ public void copyFrom(InstanceGroup instanceGroup) {
         }
     }
 
+    private void addInstances(List<String> instances) throws Exception {
+        Compute client = createComputeClient();
```

Comment:
The client could be a parameter, enabling it to be reused, as this method in called from `create` and `update` both creating their own clients.

Reply:
Done.